core-primitives/stf-executor/src/getter_executor.rs

```diff
@@ -63,11 +63,13 @@ where
 
 		trace!("Successfully decoded trusted getter");
 		if let Getter::trusted(trusted_getter_signed) = getter {
+			let getter_timer_start = Instant::now();
 			let state_result = self.state_observer.observe_state(shard, |state| {
 				StateGetter::get_state(&trusted_getter_signed, state)
 			})??;
 
 			trace!("Successfully executed trusted getter");
```

Comment:
Can you remove that trace? I don't think that's necessary any more with the debug output below.

Reply:
Yes 👍